File: C#/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.cs
using System;

namespace q{
    class w{
        static void Main(){
            string[] aa = Console.ReadLine().Split(); int ck=1;
            int[] a = Array.ConvertAll(aa,int.Parse);
            if(a[0]>a[2]){
                ck=0;
            }
            if(a[1]==a[2] && ck==1){
                Console.Write("Anything");
            }
            else if(a[2]<a[1] && ck==1){
                Console.Write("Subway");
            }
            else{
                Console.Write("Bus");
            }
        }
    }
}

File: Python/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.py
n,a,b=map(int,input().split()); ck=1
if(b<n):
    ck=0
if(b==a and ck==1):
    print("Anything")
elif(b<a and ck==1):
    print("Subway")
else:
    print("Bus")